src/Faker/ORM/CakePHP/ColumnTypeGuesser.php

```diff
@@ -25,12 +25,12 @@ public function guessFormat($column, $table)
                     return $generator->boolean;
                 };
             case 'integer':
-                return function () {
-                    return mt_rand(0, intval('2147483647'));
+                return function () use ($generator) {
+                    return $generator->numberBetween(0, 2147483647);
                 };
             case 'biginteger':
-                return function () {
-                    return mt_rand(0, intval('9223372036854775807'));
+                return function () use ($generator) {
+                    return $generator->numberBetween(0, (int) '9223372036854775807');
```

Comment:
Just write the actual integer literal here?

Reply:
I would suggest adding a comment here, too. (max 64 bit signed integer value...)